sources/packages/backend/apps/api/src/route-controllers/student/_tests_/e2e/student.institutions.controller.getStudentApplicationSummary.e2e-spec.ts

```diff
@@ -0,0 +1,221 @@
+      );
+
+      // Act/Assert
+      await request(app.getHttpServer())
+        .get(endpoint)
+        .auth(institutionUserToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.OK)
+        .expect({
+          results: [
+            {
+              id: savedApplication.id,
+              applicationNumber: savedApplication.applicationNumber,
+              studyStartPeriod:
+                savedApplication.currentAssessment.offering.studyStartDate,
+              studyEndPeriod:
+                savedApplication.currentAssessment.offering.studyEndDate,
+              applicationName: "Financial Aid Application",
+              status: savedApplication.applicationStatus,
+            },
+          ],
+          count: 1,
+        });
+    },
+  );
+
+  it(
+    "Should get the first submitted student application details as summary when student has two submitted/Inprogress " +
+      "application for the institution (application with location id saved) when pagination is 1.",
+    async () => {
+      // Arrange
+
+      // Student has one submitted and one inprogress application to the institution.
+      const student = await saveFakeStudent(appDataSource);
+      // Application 1.
+      const savedApplication1 = await saveFakeApplication(appDataSource, {
+        institution: collegeF,
+        institutionLocation: collegeFLocation,
+        student,
+      });
+
+      // Application 2.
+      await saveFakeApplication(
+        appDataSource,
+        {
+          institution: collegeF,
+          institutionLocation: collegeFLocation,
+          student,
+        },
+        { applicationStatus: ApplicationStatus.InProgress },
+      );
+
+      // By default the application is sorted by status.
+      const endpoint = `/institutions/student/${student.id}/application-summary?page=0&pageLimit=1`;
+      const institutionUserToken = await getInstitutionToken(
+        InstitutionTokenTypes.CollegeFUser,
+      );
+
+      // Act/Assert
+      await request(app.getHttpServer())
+        .get(endpoint)
+        .auth(institutionUserToken, BEARER_AUTH_TYPE)
+        .expect(HttpStatus.OK)
+        .expect({
+          results: [
+            {
+              id: savedApplication1.id,
+              applicationNumber: savedApplication1.applicationNumber,
+              studyStartPeriod:
+                savedApplication1.currentAssessment.offering.studyStartDate,
+              studyEndPeriod:
+                savedApplication1.currentAssessment.offering.studyEndDate,
+              applicationName: "Financial Aid Application",
+              status: savedApplication1.applicationStatus,
+            },
+          ],
+          count: 2,
+        });
+    },
+  );
+
+  it(
+    `Should get all the two student application details in ${FieldSortOrder.DESC} order of application number ` +
+      "as summary when student has two submitted/Inprogress application for the institution (application with location id saved) " +
```

Comment:
Just a suggestion, I would not mind omitting the explanation about "(application with location id saved)", that said I do support the long text description 😉